packages/sanity/src/core/comments/context/comments/CommentsProvider.tsx

```diff
@@ -84,7 +83,7 @@ export const CommentsProvider = memo(function CommentsProvider(props: CommentsPr
   const commentsEnabled = useCommentsEnabled()
   const [status, setStatus] = useState<CommentStatus>('open')
   const {client, createAddonDataset, isCreatingDataset} = useAddonDataset()
-  const publishedId = getPublishedId(documentId)
+  const publishedId = documentId
```

Comment:
Could it be clearer if we removed the `publishedId` binding and instead referred to `documentId` directly where it's used?